src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterfaceSecurity.cs

```diff
@@ -53,20 +57,24 @@ class ConsoleHostUserInterface : System.Management.Automation.Host.PSHostUserInt
         /// <param name="caption">Caption for the message.</param>
         /// <param name="allowedCredentialTypes">What type of creds can be supplied by the user.</param>
         /// <param name="options">Options that control the cred gathering UI behavior.</param>
+        /// <param name="confirmPassword">Prompt to confirm the password.</param>
         /// <returns>PSCredential object, or null if input was cancelled (or if reading from stdin and stdin at EOF).</returns>
 
         public override PSCredential PromptForCredential(
             string caption,
             string message,
             string userName,
             string targetName,
+            bool confirmPassword,
             PSCredentialTypes allowedCredentialTypes,
             PSCredentialUIOptions options)
         {
             PSCredential cred = null;
             SecureString password = null;
+            SecureString confirmedPassword = null;
```

Comment:
It is not request to change: maybe `retypedPassword` would be more clear.

Reply:
changed to `retpytedPassword`